StudCity.Frontend/src/pages/verify-email/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState }  from 'react';
+import TextField from '../../UI/fields/TextField';
+import { numberValidation } from '../../utils/validators/validators';
+import Svg from '../../components/Svg';
+
+function VerifyEmail() {
+    const [numbers, setNumbers] = useState(["", "", "", "", "", ""]);
```

Comment:
change the naming for example 'verifyNumbersArray', or ask our PM for a suggestion